src/Avalonia.Controls/Button.cs

```diff
@@ -349,7 +340,7 @@ private void StopListeningForDefault(IInputElement root)
         /// <summary>
         /// Raises the <see cref="Click"/> event.
         /// </summary>
-        private void RaiseClickEvent()
+        protected void RaiseClickEvent()
```

Comment:
We already have an `OnClick` method which the comments say "Raises the Click event` except it doesn't. We should probably fix that and merge this method into that method instead.